terraform-module/massdriver.yaml

```diff
@@ -11,6 +11,9 @@ type: "{{ type }}"
 {%- if paramsSchema.size > 0 %}
 {{ paramsSchema }}
 {% else %}
+steps:
```

Comment:
This should not be in the paramsSchema liquid template `else` statement

Reply:
Fixed